earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/build_and_test.py

```diff
@@ -45,6 +45,14 @@
 import opengee.version
 
 
+def build_swig(source_dir):
```

Comment:
IDK...  At the end of the day should we just say this script is for Windows building of Portable only.  And give a scons target for building portable when in Linux?

Reply:
I agree, but that would be quite a major refactor. For the time being I believe it makes the most sense to make those changes as a separate issue.